lib/service_provider_old.rb

```diff
@@ -1,6 +1,6 @@
 require 'fingerprinter'
 
-class ServiceProvider
+class ServiceProviderOld
```

Comment:
yup, just wanted to keep it around while I was removing the namespace collision.